cmd/snap-exec/main.go

```diff
@@ -127,10 +127,16 @@ func snapExecApp(snapApp, revision, command string, args []string) error {
 		return fmt.Errorf("cannot find app %q in %q", appName, snapName)
 	}
 
-	cmd, err := findCommand(app, command)
+	cmdAndArgs, err := findCommand(app, command)
 	if err != nil {
 		return err
 	}
+	// strings.Split() is ok here because we validate all app fields
+	// and the whitelist is pretty strict (see
+	// snap/validate.go:appContentWhitelist)
+	cmdArgv := strings.Split(cmdAndArgs, " ")
```

Comment:
Will this handle quoting that may be present there? Is three anything in golang that handles shell-aware split?